### ISSUE TITLE
Add space between map & next event on Chapter pages

### ISSUE THREAD

**Issue by Cara-Jo** [girldevelopit/gdi-new-site#166]
Right now, the space between the map and the next event is too small. It gives the appearance that the map is associated with the event AFTER the image, when it is in fact not. By increasing the margin on the bottom of the map, this will help eliminate that confusion. 

See screenshot: 
![screenshot 2015-02-02 09 07 07](https://cloud.githubusercontent.com/assets/180763/6003741/246bd9e0-aabb-11e4-824a-67b134513546.png)

**Comment by kstack7** [comment]
Do you think we should reduce the space between the RSVP and map, too? or just keep that spacing as-is an just add the space below the map? Not sure if that'll make too much white space going on...

**Comment by Roenok** [comment]
Reduce the space between RSVP and the map, I think. We need to make it clear that the map belongs to that event.

**Comment by kstack7** [comment]
Yeah I agree with that. It was getting too out in the middle on it's own.

**Comment by kstack7** [comment]
![screen shot 2015-02-04 at 9 49 23 pm](https://cloud.githubusercontent.com/assets/6612770/6054150/bb4192d0-acb7-11e4-85f9-247489fa568d.png)

Does this make more sense? I just moved the map below the time/address so that the RSVP can immediately follow the description. The RSVP button has a 55px bottom margin thus I think it should be at the very end to help space between the next events. Any takers? I can push this if so! :+1:

**Comment by Roenok** [comment]
@kstack7 How does it look on mobile view? I am okay with it if the map doesn't push the descriptions too far off the screen

**Comment by kstack7** [comment]
I didn't change any of the div structures or CSS so I'd assume it'll just rearrange/stack as nicely as it does now on the heroku version. The map and the description are both in their own p tags, which should keep them stacking nicely (we could always remove the float otherwise but I haven't seen anything go crazy yet).

Here's how it looks when resizing on desktop:

![screen shot 2015-02-04 at 9 54 25 pm](https://cloud.githubusercontent.com/assets/6612770/6054307/624bfd62-acb9-11e4-89cb-6f5299d8bd54.png)

**Comment by Roenok** [comment]
@kstack7 I like it! Let's pull the trigger for now.

**Comment by kstack7** [comment]
Gladly :)

**Comment by kstack7** [comment]
PR #177
